### PR TITLE
Document Update

### PR DESCRIPTION
**Pull request by NiteshKoushik**
Change in keyword exist - exists and failure to failures

### CODE REVIEW

**Comment by NiteshKoushik** [comment]
These changes are not present in Javadoc

File: docs/modules/ROOT/pages/spring-boot.adoc [hunk 1]
```diff
@@ -148,7 +148,7 @@ The component supports 157 options, which are listed below.
 | *camel.health.config.parent* | The id of the health check such as routes or registry (can use * as wildcard) |  | String
 | *camel.health.config.pool-size* | Sets the core pool size (threads to keep minimum in pool) |  | Integer
 | *camel.health.config.rejected-policy* | Sets the handler for tasks which cannot be executed by the thread pool. |  | ThreadPoolRejectedPolicy
-| *camel.health.config.time-unit* | Sets the time unit used for keep alive time |  | TimeUnit
+| *camel.health.config.time-unit* | Sets the time unit used for keeping alive time |  | TimeUnit
```
Comment (jamesnetherton) [inline docs/modules/ROOT/pages/spring-boot.adoc:151]: We need to be a bit careful about rewording some of the terminology. In this context, `keep alive time` has a specific meaning. `keeping alive time` does not. So we should stick with the original wording IMO.

Comment (NiteshKoushik) [reply docs/modules/ROOT/pages/spring-boot.adoc:151]: got it, thanks for the information. I will revert it back.

File: docs/modules/ROOT/pages/spring-boot.adoc [hunk 2]
```diff
@@ -132,14 +132,14 @@ The component supports 157 options, which are listed below.
 | *camel.component.properties.ignore-missing-location* | Whether to silently ignore if a location cannot be located, such as a properties file not found. | false | Boolean
 | *camel.component.properties.initial-properties* | Sets initial properties which will be used before any locations are resolved. The option is a java.util.Properties type. |  | String
 | *camel.component.properties.location* | A list of locations to load properties. You can use comma to separate multiple locations. This option will override any default locations and only use the locations from this option. |  | String
-| *camel.component.properties.override-properties* | Sets a special list of override properties that take precedence and will use first, if a property exist. The option is a java.util.Properties type. |  | String
+| *camel.component.properties.override-properties* | Sets a special list of override properties that take precedence and will use first, if a property exists. The option is a java.util.Properties type. |  | String
```
Comment (oscerd) [inline docs/modules/ROOT/pages/spring-boot.adoc:135]: This needs to be done in the javadoc

**Review by oscerd — CHANGES_REQUESTED** [review]
The content of this table is coming from javadoc and classes.

File: docs/modules/ROOT/pages/spring-boot.adoc (hunk 2 above)
Comment (NiteshKoushik) [reply docs/modules/ROOT/pages/spring-boot.adoc:135]: I think I have already made all the changes in the below link, this link was shared by you.
https://github.com/apache/camel-spring-boot/edit/master/core/camel-spring-boot/src/main/java/org/apache/camel/spring/boot/CamelConfigurationProperties.java

The changes which are made in this PR are not present in the javadoc file

**Comment by NiteshKoushik** [comment]
I have already made all the changes in the below link, this link was shared by you.
https://github.com/apache/camel-spring-boot/edit/master/core/camel-spring-boot/src/main/java/org/apache/camel/spring/boot/CamelConfigurationProperties.java

The changes which are made in this PR are not present in the javadoc file

**Comment by NiteshKoushik** [comment]
Any changes required @oscerd , please let me know.

**Comment by oscerd** [comment]
I need to double check, the stuff is always autogenerated so modified only there will be override. Also this Pr is dirty.. maybe open a new one and add just one commit.. you'll need to have a full build of camel and then build camel-spring boot

**Comment by NiteshKoushik** [comment]
Sure.. I can do that. I will close this PR.

**Comment by NiteshKoushik** [comment]
Created a new PR "https://github.com/apache/camel-spring-boot/pull/177" with one commit.

**Pull request closed without merging**